playbooks/redhat.yml

```diff
@@ -26,4 +26,4 @@
       tags:
         - edpm_bootstrap
       vars:
-        rhc_organization: "{{ rhc_organization | int | default(omit) }}"  # noqa: jinja[invalid]
+        rhc_organization: "{{ rhc_organization | default(omit) }}"  # noqa: jinja[invalid]
```

Comment:
It wasn't that rhc_organization was a string, it's that it was coming across as a float from the generated inventory in the k8s Secret. The commit that added "| int" was:
https://github.com/openstack-k8s-operators/edpm-ansible/commit/8bd53873ef86aff19aa6eb0a6e5f1a80487ef58d

and the issue that details the problem: https://issues.redhat.com/browse/OSPRH-15097

Instead of just dropping the filter to int, I think we'll need to add some conditional tasks to do it only when rhc_organization is defined.